Makefile

```diff
@@ -15,6 +15,7 @@ GOFMT_FILES?=$$(find . -name '*.go' | grep -v pb.go | grep -v vendor)
 
 
 GO_VERSION_MIN=1.19.1
+PROTOC_VERSION_MIN=3.21.5
```

Comment:
Should a user's protoc match exactly to the same protoc version as noted here and not just the minimum? Cause auto generated file contain that version

Reply:
Fixed in https://github.com/hashicorp/vault/pull/17122/commits/c9996bc54ed22e569a8e71273c9fe91bcb585737